package/network/utils/iwinfo/patches/0003-fix-Wduplicate-decl-specifier-warnings.patch

```diff
@@ -0,0 +1,88 @@
+From baa00cfe96ba45da59ed11009a96bee5ad702ae6 Mon Sep 17 00:00:00 2001
+From: Andre Heider <a.heider@gmail.com>
+Date: Wed, 23 Nov 2022 14:11:58 +0100
+Subject: [PATCH 03/39] fix -Wduplicate-decl-specifier warnings
+
+include/iwinfo.h:209:19: warning: duplicate 'const' declaration specifier [-Wduplicate-decl-specifier]
+  209 | static const char const *ht_secondary_offset[4] = {
+      |                   ^~~~~
+
+Constify and move the offenders out of the header.
+
+Signed-off-by: Andre Heider <a.heider@gmail.com>
+---
+ include/iwinfo.h | 28 +++++++++-------------------
+ iwinfo_lib.c     | 18 ++++++++++++++++++
+ 2 files changed, 27 insertions(+), 19 deletions(-)
+
+diff --git a/include/iwinfo.h b/include/iwinfo.h
+index 576632d..648b029 100644
+--- a/include/iwinfo.h
++++ b/include/iwinfo.h
+@@ -199,25 +199,15 @@ struct iwinfo_scanlist_vht_chan_entry {
+ 	uint8_t center_chan_2;
+ };
+ 
+-static const char *ht_secondary_offset[4] = {
+-	"no secondary",
+-	"above",
+-	"[reserved!]",
+-	"below",
+-};
+-
+-
+-static uint16_t ht_chan_width[2] = {
+-	20, /* 20 MHz */
+-	2040, /* 40 MHz or higher (refer to vht if supported) */
+-};
+-
+-static uint16_t vht_chan_width[] = {
+-	[0] = 40, /* 40 MHz or lower (refer to ht to a more precise width) */
+-	[1] = 80, /* 80 MHz */
+-	[3] = 8080, /* 80+80 MHz */
+-	[2] = 160, /* 160 MHz */
+-};
++extern const char *ht_secondary_offset[4];
++/* 0 = 20 MHz
++   1 = 40 MHz or higher (refer to vht if supported) */
++extern const uint16_t ht_chan_width[2];
++/* 0 = 40 MHz or lower (refer to ht to a more precise width)
++   1 = 80 MHz
++   2 = 160 MHz
++   3 = 80+80 MHz */
++extern const uint16_t vht_chan_width[4];
+ 
+ struct iwinfo_scanlist_entry {
+ 	uint8_t mac[6];
+diff --git a/iwinfo_lib.c b/iwinfo_lib.c
+index 22571f0..ab74591 100644
+--- a/iwinfo_lib.c
++++ b/iwinfo_lib.c
+@@ -78,6 +78,24 @@ const char *IWINFO_HTMODE_NAMES[] = {
+ 	"HE160"
+ };
+ 
++const char *ht_secondary_offset[4] = {
```

Comment:
Please make the chars also const. 
```suggestion
+const char * const ht_secondary_offset[4] = {
```

Reply:
Yeah, I noticed that too but didn't do it in the end. I guess consumers will get "discards qualifier" warnings then, which may break builds.
But lemme try if it actually does...

Reply:
Done, but rpcd builds with `-Werror` and thus failed with "discards qualifier". I added a rpcd patch for that, but now we have a dependency, so if we keep it like this both have to be commited or we have a broken build. That's rest builds fine though, at least for my config. Let's see if CI finds more spots...